astro/src/content/docs/apis/_user-response-body-base.mdx

```diff
@@ -385,11 +385,6 @@ subsequent login requests to bypass the Two Factor challenge.
   </DeprecatedSince>
 </APIField>
 
-<APIField name={props.base_field_name + '.verifiedInstant'} type="Long" since="1.48.0" deprecated>
-  The [instant](/docs/reference/data-types#instants) that the User's primary email address was verified. This does NOT indicate when phone number identity types
-  were verified.
-
-  <DeprecatedSince since="1.99.9">
-    See <InlineField>{props.base_field_name}.identities\[x].verifiedInstant</InlineField>.
-  </DeprecatedSince>
+<APIField name={props.base_field_name + '.verifiedInstant'} type="Long" since="1.48.0">
+  The first [instant](/docs/reference/data-types#instants) that the User's primary email address or phone number was verified. This value is immutable and cannot change for a user once set.
```

Comment:
skipped over the nuance of phone vs. email here

Reply:
For details on what that nuance is:
1. With email identities in this code (and legacy FusionAuth), `user.verifiedInstant` gets set even if verification is skipped or disabled.
2. For phone identities, I chose not to do that. A phone identity only affects `user.verifiedInstant` if verification was actually completed (implicit or completed).
3. For both phone and email, the identity level `verifiedInstant` field is only set, in 1.99.9, if verification was actually completed (implicit or completed).